test/testcafe/framework/page-objects/FactorEnrollPasswordPageObject.js

```diff
@@ -69,7 +69,7 @@ export default class EnrollPasswordPageObject extends BasePageObject {
     const expectedTitleValue = expectComplete ? 'complete' : 'incomplete';
     const passwordMatchWrapper = this.form.getElement('[data-se="password-authenticator--matches');
 
-    return within(passwordMatchWrapper).queryByRole('img', { name: expectedTitleValue }).exists;
+    return within(passwordMatchWrapper).findByTitle(expectedTitleValue).exists;
```

Comment:
Is this change to `findByTitle` necessary because now that we use `aria-hidden` the element no longer is picked up but `role=img`?

Reply:
yup that is exactly right